docs/ballot.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Ballot Operation
+id: ballot
+author: Davis Sawali
+---
+
+
+The `Ballot` operation allows delegates to cast one `Yay`, `Nay`, or `Pass` ballot on a selected proposal. Delegates are only able to cast their votes during the **Exploration period**.
+
+
+## Examples
+The `Ballot` operation is currently available in the Contract API, and can be used as such:
+```typescript
+const op = await Tezos.contract.ballot({
+  period: 1,
+  proposal: 'PROPOSAL_HASH',
+  ballot: 'BALLOT_VOTE_STRING'
+});
+
+await op.confirmation();
+```
+- `period` is the voting period of the current protocol
+- `proposal` is the proposal hash string that you (a delegate) would like to point your ballot towards
```

Comment:
Can you explain in the doc how to find this proposal hash? (using rpcClient.getCurrentProposal)

Reply:
good point, will add